_posts/blog/2019-08-01-why-upgrade.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Why Upgrade to Accumulo 2.0"
+author: Mike Miller
+---
+
+Accumulo 2.0 has been in development for quite some time now and is packed with new features, bug
+fixes, performance improvements and redesigned components.  All of these changes bring challenges
+when upgrading your production cluster so you may be wondering... why should I upgrade?
+
+My top 10 reasons to upgrade. For all changes see the [release notes][rel]
+
+* [Summaries](#summaries)
+* [New Bulk Import](#new-bulk-import)
+* [Simplified Scripts and Config](#simplified-scripts-and-config)
+* [New Monitor](#new-monitor)
+* [New Accumulo Client](#new-accumulo-client)
+* [Hadoop 3](#hadoop-3-support)
+* [Offline creation](#offline-creation)
+* [Search Documentation](#search-documentation)
+* [Java 11 Support](#java-8-11-support)
```

Comment:
1.9 also supports Java 11, so its not exclusive to 2.0

Reply:
1.9.3 and earlier did not support Java 11. 1.9.4 will be the first in that series to seriously try to support it. So, currently, it might actually be exclusive to 2.0.

Reply:
I could replace the Java or hadoop bullet with something more compelling and unique to 2.0, like the Scan Executors.